docs/api/protocol.md

```diff
@@ -1,7 +1,6 @@
 # protocol
 
-The `protocol` module can register a custom protocol or intercept an existing
-protocol.
+> Register a custom protocol and intercept existing protocols.
```

Comment:
Do you think `intercept existing protocol requests` might be clearer than `intercept existing protocols`?